Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -2416,7 +2420,8 @@ public ReferenceDescriptionCollection FetchReferences(NodeId nodeId)
             // save session id.
             lock (SyncRoot)
             {
-                SessionCreated(sessionId, sessionCookie);
+                // save session id and cookie in base
```

Comment:
revert to previous behavior, but do the calls with virtual callback at the end when the session is ready to use..